src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,14 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms transform, 300ms color;
```

Comment:
```suggestion
    transition: transform 300ms , color 300ms ;
```

Reply:
It is better to put transition property name first then other attributes.
You can read more about it [here](https://developer.mozilla.org/en-US/docs/Web/CSS/transition)